mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -1,9 +1,19 @@
 CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 	METHOD method_17884 (Ljava/util/Map;Lnet/minecraft/class_2487;Lnet/minecraft/class_1887;)V
 		ARG 2 enchantment
-	METHOD method_22445 getEnchantments (Lnet/minecraft/class_2499;)Ljava/util/Map;
+	METHOD method_22445 fromTag (Lnet/minecraft/class_2499;)Ljava/util/Map;
+		COMMENT Loads enchantments from a list named binary tag.
 		ARG 0 tag
-	METHOD method_24365 getMatchingEnchantedEquipment (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;Ljava/util/function/Predicate;)Ljava/util/Map$Entry;
+	METHOD method_24365 getRandomEnchantedEquipment (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;Ljava/util/function/Predicate;)Ljava/util/Map$Entry;
```

Comment:
I feel like "Random" shouldn't be part of the name here, as it might be an implementation detail. I would rather go with something like `getEquipmentEnchantedWith`, while keeping the new javadocs.

Reply:
how about getRandom -> choose?

Reply:
Renamed. The new name is `chooseEquipmentWith`